lib/b2b/sso.ts

```diff
@@ -32,93 +30,207 @@ export interface SAMLConnection {
   signing_certificates: X509Certificate[];
   verification_certificates: X509Certificate[];
+  attribute_mapping?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
 }
 
-export interface B2BSSOGetConnectionsRequest {
-  organization_id: string;
-}
-
-export interface B2BSSOGetConnectionsResponse extends BaseResponse {
-  saml_connections: SAMLConnection[];
-  oidc_connections: OIDCConnection[];
-}
-
-export interface B2BSSODeleteConnectionRequest {
-  organization_id: string;
-  connection_id: string;
-}
-
-export interface B2BSSODeleteConnectionResponse extends BaseResponse {
-  connection_id: string;
+export interface X509Certificate {
+  certificate_id: string;
+  certificate: string;
+  issuer: string;
+  created_at?: Date;
+  expires_at?: Date;
 }
 
+// Request type for `SSO.authenticate`.
 export interface B2BSSOAuthenticateRequest {
+  // The token to authenticate.
   sso_token: string;
+  // A base64url encoded one time secret used to validate that the request starts and ends on the same device.
+  pkce_code_verifier?: string;
+  // The `session_token` belonging to the member that you wish to associate the email with.
   session_token?: string;
+  // The `session_jwt` belonging to the member that you wish to associate the email with.
   session_jwt?: string;
+  /**
+   * Set the session lifetime to be this many minutes from now. This will start a new session if one doesn't
+   * already exist,
+   *   returning both an opaque `session_token` and `session_jwt` for this session. Remember that the
+   * `session_jwt` will have a fixed lifetime of
+   *   five minutes regardless of the underlying session duration, and will need to be refreshed over time.
+   *
+   *   This value must be a minimum of 5 and a maximum of 527040 minutes (366 days).
+   *
+   *   If a `session_token` or `session_jwt` is provided then a successful authentication will continue to
+   * extend the session this many minutes.
+   *
+   *   If the `session_duration_minutes` parameter is not specified, a Stytch session will be created with a
+   * 60 minute duration. If you don't want
+   *   to use the Stytch session product, you can ignore the session fields in the response.
+   */
   session_duration_minutes?: number;
+  /**
+   * Add a custom claims map to the Session being authenticated. Claims are only created if a Session is
+   * initialized by providing a value in
+   *   `session_duration_minutes`. Claims will be included on the Session object and in the JWT. To update a
+   * key in an existing Session, supply a new value. To
+   *   delete a key, supply a null value. Custom claims made with reserved claims (`iss`, `sub`, `aud`,
+   * `exp`, `nbf`, `iat`, `jti`) will be ignored.
+   *   Total custom claims size cannot exceed four kilobytes.
+   */
   session_custom_claims?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
-  pkce_code_verifier?: string;
-  locale?: "en" | "es" | "pt-br";
```

Comment:
`locale` shouldn't be removed - this is part of the MFA changes but it's not feature flagged or anything, so I'm not sure why it was removed. It was added more recently so maybe just need to run the codegen again